utils/installer/update_lvim.sh

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env bash
+
+LUNARVIM_RUNTIME_DIR="${LUNARVIM_RUNTIME_DIR:-"$HOME/.local/share/lunarvim"}"
```

Comment:
Since these variables are 'defined' when running the setup script, maybe we should instead generate this file

Reply:
I was able to remove that file entirely now and implement the functionality in Lua.